queue/src/fiskaltrust.Middleware.Contracts/RequestCommands/RequestCommand.cs

```diff
@@ -17,7 +17,7 @@ public abstract class RequestCommand
 
         protected ReceiptResponse CreateReceiptResponse(ftQueue queue, ReceiptRequest request, ftQueueItem queueItem, string ftCashBoxIdentification, long ftState)
         {
-            var receiptIdentification = $"ft{queue.ftReceiptNumerator:X}#";
+            var receiptIdentification = $"ft{queue.ftReceiptNumerator:X}";
```

Comment:
```suggestion
            var receiptIdentification = $"ft{queue.ftReceiptNumerator:X}#";
```
Think this was a typo 😅